packages/expo-sqlite/build/SQLite.js

```diff
@@ -56,7 +56,9 @@ function _escapeBlob(data) {
 }
 const _openExpoSQLiteDatabase = customOpenDatabase(SQLiteDatabase);
 function addExecMethod(db) {
-    db.exec = db._db.exec;
+    db.exec = (queries, readOnly, callback) => {
```

Comment:
I swear I didn't add this, it must be my typescript compiler

Reply:
Or a not up-to-date JS 🙃